tokio-trace/test_static_max_level_features/main.rs

```diff
@@ -0,0 +1,75 @@
+#[macro_use]
+extern crate log;
+
+use std::sync::{Arc, Mutex};
+use log::{Level, LevelFilter, Log, Record, Metadata};
```

Comment:
we'll want to change this to import types from `tokio-trace`:
```suggestion
use tokio_trace::{field, span, Level, LevelFilter, Event, Metadata. Subscriber};
```